### ISSUE TITLE
Storybook: Docs tab code examples are broken in React Storybook

### ISSUE THREAD

**Issue by daneah** [ithaka/pharos#515]
**Expected behavior**

Consumers can view the **Docs** tab in Storybook and expand the **Show code** sections to view the code that generated the corresponding story.

**Actual behavior**

The **Show code** section ends up showing what looks like some kind of internal Storybook representation of the story, e.g.:

```jsx
<[object Object]
  value={{
    prefix: 'storybook'
  }}
>
  <No Display Name />
</[object Object]>
```

**Steps to reproduce the issue**

1. Visit any story in the **React** section of the combined Storybook or in the standalone React Storybook
2. Click the **Docs** tab
4. On any story, click **Show code**

**Additional information**

This likely broke as a result of #501 but I couldn't readily confirm because we don't have historical record of the React Storybook—only the web component Storybook. It would be easy enough (though possibly slow) to confirm locally by going back a few commits (or with `git bisect` if this turns out to be older).

**Comment by daneah** [comment]
Fixed as a side effect of #543!

**Comment by daneah** [comment]
Fixed as a side effect of #543!
